sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Tests/Azure.Monitor.OpenTelemetry.AspNetCore.Tests.csproj

```
@@ -35,7 +35,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" VersionOverride="6.0.21" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net7.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" VersionOverride="7.0.9" />
   </ItemGroup>
```

```suggestion
  <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
    <FrameworkReference Include="Microsoft.AspNetCore.App" />
    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" VersionOverride="8.0.7" />
  </ItemGroup>
```

Need to also update the package version. These packages are tightly coupled to the framework version.
https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Testing/8.0.7#supportedframeworks-body-tab

---

If this condition could instead be moved to the Packages.Data.props, that would save me one headache. :)

https://github.com/Azure/azure-sdk-for-net/blob/6188d305fa61e8a09100a469177c926789f83d0b/eng/Packages.Data.props#L284

---

I opened an issue for it! I have a few more package modernization updates so will try to get it in with those https://github.com/Azure/azure-sdk-for-net/issues/45426